interface/forms/LBF/new.php

```diff
@@ -1525,21 +1524,16 @@ function warehouse_changed(sel) {
                             echo "&nbsp;&nbsp;";
                         }
                         ?>
-                        <button type="submit" class="btn btn-secondary btn-save" name="bn_save"
-                                value="<?php echo xla('Save'); ?>">
+                        <button type="submit" class="btn btn-primary btn-save" name="bn_save">
```

Comment:
This button now just clears the form and then doesn't do anything

Reply:
Here, I only change the button style. Is it broken on the master branch?

Reply:
hi @stu01509 , I just remembered the reason for this issue, since I also experienced this in the past. The button submit needs to keep the html `value` attribute since that is sent back with the form in this case as:
name of parameter:
`bn_save`
value of bn_save parameter:
`xla('Save')`

So, if you remove the `value`, then the bn_save parameter will be empty, which then throws off the submission and breaks the script. So, just need to keep the html `value` attribute there; note this is also the reason your prior input->button changes were failing. It is worthwhile to change input to button since this then allows the cool fontawesome class icons to work (just keep the `value` html also set in those cases also).

Reply:
> value of bn_save parameter:
> `xla('Save')`

Be careful translating a button value. Because a translated submitted Save may be missed by the non translated value in save conditional in script. eg if('Save') then ....
Best to use translates Save as `<button>xlt('Save')</button>`